getting-started/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Copyright 2017 Google Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+    <threadsafe>true</threadsafe>
+    <runtime>java8</runtime>
+    <system-properties>
+        <property name="java.util.logging.config.file" value="WEB-INF/logging.properties"/>
+    </system-properties>
```

Comment:
This is where we can add an environment variable here, e.g

```xml
<env-variables>
  <env-var name="GOOGLE_STORAGE_BUCKET" value="your-storage-bucket" />
</env-variables>
```

See [the java appengine-8 samples](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/e38719e7b18d8b50bde848d639dbea39e1fa83e2/appengine-java8/pubsub/src/main/webapp/WEB-INF/appengine-web.xml) for an example in code.

Reply:
this can also go into applications.properties file, and can be overridden via a number of ways.